src/libmongoc/src/mongoc/mongoc-error-private.h

```diff
@@ -45,11 +45,14 @@ typedef enum {
    MONGOC_SERVER_ERR_EXCEEDEDTIMELIMIT = 262,
    MONGOC_SERVER_ERR_SOCKETEXCEPTION = 9001,
    MONGOC_SERVER_ERR_NOTMASTER = 10107,
```

Comment:
Do we still need these legacy codes in the enum?

Reply:
You are correct, this enum is not part of the public API. I removed the flags.